apps/docs/content/guides/ai/rag-with-permissions.mdx

```diff
@@ -189,7 +189,7 @@ Since we're managing users and authentication outside of Supabase, we have two o
 
 #### Direct Postgres connection
 
-You can directly connect to your Supabase Postgres DB using the [connection info](/dashboard/project/_/database/settings) on your project's Database Settings page. To use RLS with this method, we use a custom session variable that contains the current user's ID:
+You can directly connect to your Supabase Postgres DB using the [connection info](/dashboard/project/_/?showConnect=true) on a project page. To use RLS with this method, we use a custom session variable that contains the current user's ID:
```

Comment:
I noticed we’re using absolute URLs elsewhere in the docs, and wonder if we should do the same here:

```suggestion
You can directly connect to your Supabase Postgres DB using the [connection info](https://supabase.com/dashboard/project/_/?showConnect=true) on a project page. To use RLS with this method, we use a custom session variable that contains the current user's ID:
```

Just in case someone accesses these docs not via our frontend, or if we ever change where docs are hosted.